README.md

```diff
@@ -87,7 +87,6 @@ Components in beta stage may include breaking changes, new features, and bug fix
 | [terra-i18n](https://github.com/cerner/terra-core/tree/master/packages/terra-i18n) | [![NPM version](https://badgen.net/npm/v/terra-i18n)](https://www.npmjs.org/package/terra-i18n) | ![Stable](https://badgen.net/badge/status/Stable/green) | [![terra-i18n](https://badgen.net/david/dep/cerner/terra-core/packages/terra-i18n)](https://david-dm.org/cerner/terra-core?path=packages/terra-i18n) |
 | [terra-icon](https://github.com/cerner/terra-core/tree/master/packages/terra-icon) | [![NPM version](https://badgen.net/npm/v/terra-icon)](https://www.npmjs.org/package/terra-icon) | ![Stable](https://badgen.net/badge/status/Stable/green) | [![terra-icon](https://badgen.net/david/dep/cerner/terra-core/packages/terra-icon)](https://david-dm.org/cerner/terra-core?path=packages/terra-icon) |
 | [terra-image](https://github.com/cerner/terra-core/tree/master/packages/terra-image) | [![NPM version](https://badgen.net/npm/v/terra-image)](https://www.npmjs.org/package/terra-image) | ![Stable](https://badgen.net/badge/status/Stable/green) | [![terra-image](https://badgen.net/david/dep/cerner/terra-core/packages/terra-image)](https://david-dm.org/cerner/terra-core?path=packages/terra-image) |
-| [terra-legacy-theme](https://github.com/cerner/terra-core/tree/master/packages/terra-legacy-theme) | [![NPM version](https://badgen.net/npm/v/terra-legacy-theme)](https://www.npmjs.org/package/terra-legacy-theme) | ![Stable](https://badgen.net/badge/status/Stable/green) | [![terra-legacy-theme](https://badgen.net/david/dep/cerner/terra-core/packages/terra-legacy-theme)](https://david-dm.org/cerner/terra-core?path=packages/terra-legacy-theme) |
```

Comment:
This needs move to the deprecated section: https://github.com/cerner/terra-core/blob/a6fbda25514fd8d07875907af2bbe31f43d7250f/README.md#--deprecated

Reply:
Updated

Reply:
Does "npm run docs" automatically update this readme?